tests/integration_tests/functional/test_signals.py

```diff
@@ -58,7 +58,19 @@ def test_generic_signal_handler(test_microvm_with_api, signum):
     assert len(line_metrics) == 1
 
     os.kill(firecracker_pid, signum)
-    msg = 'Shutting down VM after intercepting signal {}'.format(signum)
+
+    # Firecracker gracefully handles SIGPIPE (doesn't terminate).
+    # False positive during lint - object member is created dynamically.
```

Comment:
ultra-nit: to get rid of this, i think you can do a cast to int(). As far as I've seen it works.
Keeping it this way is fine as well

Reply:
Thanks, didn't know that would work.